index.js

```diff
@@ -1,6 +1,41 @@
 /* jshint node: true */
 'use strict';
 
+var path = require('path');
+var mergeTrees = require('broccoli-merge-trees');
+
 module.exports = {
-  name: 'ember-redux'
+  name: 'ember-redux',
+
+  included: function(app) {
+    // Import UMD redux and redux-thunk
```

Comment:
Right about users not removing the prefix getting a bigger bundle.  I wonder how many users are importing redux in their own code.

I think if we are to go down the road of programmatically checking packages, it should be on the 1.X series with a deprecation notice—then full removal in 2.0 with clear release notes.

@toranb I can submit a separate pull request to be included in 1.X with a deprecation notice and then rebase this PR against it for 2.0, but what do you think?